src/select/select.module.scss

```
@@ -225,3 +231,26 @@
     background-color: $slate-20;
   }
 }
+
+.input-search-wrap {
+  position: absolute;
+  width: 100%;
+  z-index: 999;
+  background: $white;
+  box-shadow: 0 2px 6px 0 rgba(41, 70, 97, 0.2);
+
+  :global(.search__control) {
+    margin: 18px 18px 22px;
```

Let's make these variables. If we can drop the bottom-margin by 2px also, we can use a variable for that.

```suggestion
    margin: $scale-base-up-03 $scale-base-up-03 $scale-base-up-04;
```